.github/actions/environment/action.yml

```diff
@@ -52,4 +52,7 @@ runs:
         if ${{ ! startsWith(github.ref, 'refs/tags/') }}; then
             git tag -d "$(git tag --points-at HEAD)" 2>/dev/null || true
         fi
+        # Found that action/checkout does not remove all temporary
+        # files from previous runs, to avoid of it need to run:
+        git submodule foreach git clean -xffd
```

Comment:
AFAIR, quotes around `git clean -xfd` are required on some git versions.

Reply:
Ok, sure, let's add it.